frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -8,26 +8,27 @@
 
 @RestController
 public class CarController {
-
-    private List<Car> currentCarList = new ArrayList<>();
     Map<Integer, Car> map = new HashMap<>();
 
     @PostMapping(path = "/car", consumes = {"application/json"})
-    public List<Car> addCar(@RequestBody Car car) {
-        currentCarList.add(car);
-        return currentCarList;
+    public Map<Integer,Car> addCar(@RequestBody Car car) {
+        map.put(map.size()+1,car);
+        car.setId(map.size());
```

Comment:
This line, at the moment, is redundant. You are putting the car into the map with the key `map.size()+1` which works but isn't an ideal way to express the relationship between the key and the value.  Secondly, you are then setting the ID of the car that's passed in via the parameter (doesn't it need an ID anyways in the request?) after you put it in the map. So this car with the new id is never used.